stylix/target.nix

```
@@ -2,14 +2,16 @@
 
 with lib;
 
-let
-  fromOs = import ./fromos.nix { inherit lib args; };
-in {
-  options.stylix.autoEnable = mkOption {
-    description = "Whether to automatically enable styling for installed targets.";
-    type = types.bool;
-    default = fromOs [ "autoEnable" ] true;
-  };
+{
+  options.stylix.autoEnable = let
+    fromOs = import ./fromos.nix { inherit lib args; } [ "autoEnable" ];
+  in
+    mkEnableOption
+    "styling installed targets"
+    // {
+      default = fromOs true;
+      example = fromOs false;
```

`fromOs` isn't necessary for examples. It exists to take the current value from the NixOS configuration and use it as the default in the Home Manager configuration. The second argument specifies what the default should be when this is the NixOS configuration, or when there is no NixOS configuration to copy from.

```suggestion
      example = false;
```

---

Closed by https://github.com/danth/stylix/pull/398/commits/4a52e418b2aa229ef98e21da229e138810a0c36b.